ckanext/archiver/plugin.py

```diff
@@ -227,8 +227,7 @@ class TestIPipePlugin(p.SingletonPlugin):
     """
     p.implements(IPipe, inherit=True)
 
-    def __init__(self, *args, **kwargs):
-        self.calls = []
+    calls = []
```

Comment:
how does that make it pass :/

Reply:
It's mentioned in the commit message. Basically, overriding `__init__` causes the superclass `__init__` not to be called.